xml/2018-10-01.xml

```diff
@@ -0,0 +1,232 @@
+    and EIE students who are automatically members of DoCSoc at no cost. For
+    those _not_ on these courses, membership can be purchased on the [Union website](https://www.imperialcollegeunion.org/activities/a-to-z/computing).
+    Why not get your friends on other courses to become members too?
+
+    **Keeping Updated -** Our weekly digest is one of two key ways to find out about
+    events. The best place to stay updated about events is via our **[Facebook](https://www.facebook.com/ICDocSoc/)**
+    page. You can also find us on [Twitter](https://twitter.com/icdocsoc/).
+
+    Don't forget that we run **group chats for every academic year** on Facebook
+    Messenger. Other than Piazza (run by the department), Messenger is a central
+    place for discussions so if you aren't part of this group chat, get in touch
+    with us and we'll add you!
+  </text>
+
+  <hr />
+
+  <heading>Volunteering with DoCSoc</heading>
+  <text>
+    Running activities at DoCSoc's scale would not be possible without the help
+    of volunteers. We have opportunities open for anyone looking to offer their
+    technical skills or manpower.
+
+    **Operations -** DoCSoc has a central [Facebook group](https://www.facebook.com/groups/1509850436007222/)
+    for when we need volunteers. If you're thinking of volunteering, join the group!
+    Events we will need help for over the coming months include:
+
+    - Google DevFest London - We're hosting Europe's largest free developer conference
+    with over 1000 attendees on Saturday 1st December. We're looking for volunteers
+    to help with organising both before and during the event.
+
+    - IC Hack - With 300 attendees, our yearly hackathon won't be possible without
+    our volunteers. If you aren't looking to compete in the hackathon, lending a
+    helping hand is also a great way to participate in the hackathon!
+
+    **Software Foundation -** We're reviving our open source initiative, the
+    [Software Foundation](https://github.com/docsocsf) and are in need of
+    volunteers and contributors. This year, we're planning lots of projects,
+    from beginner-friendly to more advanced. If you'd like to get involved,
+    either as a project owner or only to suggest ideas, please let us know!
+  </text>
+
+  <hr />
+
+  <text>Now for the upcoming events:</text>
+
+  <heading>Freshers' Fair</heading>
+  <text>
+    - **When:** **TOMORROW!** 11:00 - 16:00, Tuesday 2nd October 2018
+    - **Where:** Main Entrance (ICME) Stall 5
+
+    Come and visit us at our stall at the Freshers' Fair to learn more about us,
+    chat with the committee and compete in a (yet to be announced) competition to
+    win some prizes! We'll be there all day so make sure to check out all of the
+    other fantastic opportunities as well. You can find us by consulting the official
+    map.
+  </text>
+
+  <heading>Mums &amp; Dads Lunch</heading>
+  <text>
+    - **When:** 12:30 - 14:00, Monday 8th October 2018
```

Comment:
This + others
Why not use the <event> formatting? You can create events without a header image if you don't have those at your disposal.

Reply:
In future, yes, but since this is quite a hefty digest for freshers, I wanted to make it as readable for them as possible.